composer/utils/device.py

```diff
@@ -8,7 +8,7 @@
 import torch.cuda
```

Comment:
this is a short file.. should this be in `composer.devices` instead?

Reply:
Agree, will move

Reply:
Will do in separate PR -- it causes some messiness